packages/web5/lib/src/dids/structures/service.dart

```diff
@@ -35,4 +35,12 @@ class DidService implements DidResource {
       'serviceEndpoint': serviceEndpoint,
     };
   }
+
+  factory DidService.fromJson(Map<String, dynamic> json) {
+    return DidService(
+      id: json['id'],
+      type: json['type'],
+      serviceEndpoint: json['serviceEndpoint'].toString(),
+    );
+  }
```

Comment:
Should this file be named `did_service.dart`?

Reply:
done!